addons/test_website/models/model.py

```diff
@@ -31,3 +31,23 @@ def _search_get_detail(self, website, order, options):
             'icon': 'fa-check-square-o',
             'order': 'name asc, id desc',
         }
+
+class WebsiteMailTestPortal(models.Model):
+    """ A model inheriting from mail.thread and portal.mixin with some fields
+    used for portal sharing, like a partner, ... (similar model as defined in
+    test_portal for testing with the website module) """
+    _description = 'Chatter Model for Portal'
+    _name = 'website.mail.test.portal'
+    _inherit = [
+        'portal.mixin',
+        'mail.thread',
+    ]
+
+    name = fields.Char('Name')
+    partner_id = fields.Many2one('res.partner', 'Customer')
+    user_id = fields.Many2one(comodel_name='res.users', string="Salesperson")
```

Comment:
Would leave out "comodel_name", it's not there for the partner_id field anyway.

(I think it's copied as is but we can adapt a bit ^^)